test-app/package.json

```diff
@@ -0,0 +1,96 @@
+{
+  "name": "test-app",
+  "version": "2.1.0",
+  "description": "An Ember addon that provides a Date service with a minimal API.",
+  "repository": "https://github.com/scalvert/ember-date-service",
+  "license": "MIT",
+  "author": "scalvert <steve.calvert@gmail.com>",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
+    "lint:js": "eslint .",
+    "start": "ember serve",
+    "test": "npm-run-all lint:* test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each"
+  },
+  "devDependencies": {
+    "@babel/eslint-parser": "^7.18.2",
+    "@ember/optional-features": "^2.0.0",
+    "@ember/test-helpers": "^2.8.1",
+    "@glimmer/component": "^1.0.2",
+    "@glimmer/tracking": "^1.0.2",
+    "broccoli-asset-rev": "^3.0.0",
+    "ember-auto-import": "^2.4.1",
+    "ember-cli": "^3.28.0",
+    "ember-cli-babel": "^7.22.1",
+    "ember-cli-dependency-checker": "^3.1.0",
+    "ember-cli-htmlbars": "^6.0.1",
+    "ember-cli-inject-live-reload": "^2.0.2",
+    "ember-cli-sri": "^2.1.1",
+    "ember-cli-terser": "^4.0.0",
+    "ember-date-service": "2.1.0",
+    "ember-disable-prototype-extensions": "^1.1.3",
+    "ember-export-application-global": "^2.0.0",
+    "ember-load-initializers": "^2.1.1",
+    "ember-maybe-import-regenerator": "^0.1.6",
+    "ember-qunit": "^5.1.5",
+    "ember-resolver": "^8.0.2",
+    "ember-source": "~3.22.0",
+    "ember-source-channel-url": "^3.0.0",
+    "ember-template-lint": "^4.9.1",
+    "ember-try": "^1.0.0",
+    "eslint": "^7.11.0",
+    "eslint-config-prettier": "^7.0.0",
+    "eslint-plugin-ember": "^9.3.0",
+    "eslint-plugin-node": "^11.0.0",
+    "eslint-plugin-prettier": "^3.2.0",
+    "loader.js": "^4.7.0",
+    "npm-run-all": "^4.1.5",
+    "prettier": "^2.2.1",
+    "qunit": "^2.19.1",
+    "qunit-dom": "^2.0.0",
+    "release-it": "^14.0.3",
+    "release-it-lerna-changelog": "^3.1.0",
+    "webpack": "5"
+  },
+  "engines": {
+    "node": ">= 14.*"
+  },
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org"
+  },
+  "changelog": {
+    "labels": {
+      "breaking": ":boom: Breaking Change",
+      "enhancement": ":rocket: Enhancement",
+      "bug": ":bug: Bug Fix",
+      "documentation": ":memo: Documentation",
+      "internal": ":house: Internal",
+      "generators": ":recycle: Generators"
+    }
+  },
+  "release-it": {
+    "plugins": {
+      "release-it-lerna-changelog": {
+        "infile": "CHANGELOG.md",
+        "launchEditor": true
+      }
+    },
+    "git": {
+      "tagName": "v${version}",
+      "changelog": "git log --pretty=format:\"* %s (%h)\" --no-merges --perl-regexp --author=\"^((?!dependabot-preview).*)$\" ${latestTag}...HEAD"
+    },
+    "github": {
+      "release": true,
+      "tokenRef": "GITHUB_AUTH"
+    }
+  },
```

Comment:
We're never going to release this package, so probably don't need this release-it config?